cmake/DAQ.cmake

```diff
@@ -780,8 +784,13 @@ function(daq_add_plugin pluginname plugintype)
   endif()
 
   add_library( ${pluginlibname} MODULE ${PLUGIN_PATH}/${pluginname}.cpp)
+  target_link_options( ${pluginlibname} PRIVATE "LINKER:--no-undefined") # A plugin should have all its contents defined
 
-  target_link_libraries(${pluginlibname} ${PLUGOPTS_LINK_LIBRARIES})
+  if (NOT DEFINED CETLIB OR NOT DEFINED CETLIB_EXCEPT)
```

Comment:
Isn't a check on CETLIB sufficient? CETLIB_EXCEPT is a dependency of CETLIB.
I would keep the check minima;.